frontend_vue/src/components/HistoryToken.vue

```diff
@@ -129,12 +132,17 @@ async function fetchTokenHistoryData() {
     const res = await historyToken(params);
     const historyTokenData = (await res.data) as HistoryTokenBackendType;
     hitsList.value = historyTokenData.history.hits.sort((a, b) => b.time_of_hit - a.time_of_hit);
-    
+    tokenType.value = historyTokenData.history.token_type;
+
     emits(
       'update-token-title',
       historyTokenData.history.token_type,
       route.params.token
     );
+
+    if (tokenType.value == TOKENS_TYPE.CREDIT_CARD_V2 && hitsList.value.length > 0) {
```

Comment:
I see you have a tokensWithoutGeoIPInfo in the code below.
How using such constant all along the file, instead of including an if statement?
It would make the code more readable